apps/portals/src/configurations/elportal/synapseConfigs/projects.ts

```diff
@@ -31,8 +31,8 @@ export const projectCardConfiguration: CardConfiguration = {
   titleLinkConfig: {
     isMarkdown: false,
     baseURL: 'Explore/Projects/DetailsPage',
-    URLColumnName: 'grant',
-    matchColumnName: 'grant',
+    URLColumnName: 'shortName',
+    matchColumnName: 'shortName',
```

Comment:
One grant could refer to many projects, so projects now use the `shortName` column as the key